Makefile

```diff
@@ -38,8 +38,10 @@ $(PROTO_OUT):
 
 ##### git submodule for proto files #####
 update-proto-submodule:
-	printf $(COLOR) "Update proto-submodule..."
+	printf $(COLOR) "Update api proto-submodule...\n"
 	git -c protocol.file.allow=always submodule update --init --force --remote $(PROTO_ROOT)
+	printf $(COLOR) "Update cloud proto-submodule...\n"
+	git -c protocol.file.allow=always submodule update --init --force --remote $(PROTO_CLOUD_ROOT)
```

Comment:
Assuming this actually pulls down latest submodule changes, I don't think we want this for the cloud submodule. I think we want that submodule to be updated manually as it is wanted (ideally the repo goes away soon anyways).